packages/office-ui-fabric-react/src/utilities/rtl.ts

```diff
@@ -0,0 +1,13 @@
+import { getRTL, getRTLSafeKeyCode } from './../Utilities';
+import { ITheme } from '@uifabric/styling';
+
+export function getRTLWithTheme(theme?: ITheme): boolean {
```

Comment:
Couldn't we just add an optional `theme` argument to the existing `getRTL`?

Reply:
I considered it, but I don't want to have the utilities package take a dependency on styling.

Reply:
Makes sense. You could get around that by using a "theme-like" object `theme?: { rtl?: boolean }`, though that's not ideal either.

Reply:
I expected what @ecraig12345 suggested. I was going to look at fixing a shimmer issue, and thought that this PR might have been merged so I tried just passing theme into `getRTL` and it didn't work.

I would prefer that we just have 1 API in utilities; it's weird to have 2.